kontrol-frontend/src/components/CytoscapeGraph/plugins/tippy.ts

```diff
@@ -29,15 +30,28 @@ const tippyFactory: cytoscapePopper.PopperFactory = (ref, content) => {
 export const createTooltip = (
   node: cytoscape.NodeSingular,
 ): Instance | null => {
-  const versions = node.data("versions");
+  const versions: NodeVersion[] = node.data("versions");
+  console.log("versions", versions);
   if (!versions || versions.length === 0) {
     return null;
   }
   const tip = node.popper({
     content: () => {
       const elem = document.createElement("div");
       // TODO: sanitize
-      elem.innerHTML = `Versions: <ul>${versions.map((v: string) => `<li>${v.toString()}</li>`).join("")}</ul>`;
+      elem.innerHTML = `
+      <table>
+        <thead>
+          <tr>
+            <th>Flow ID</th>
+            <th>Image Tag</th>
+          </tr>
+        </thead>
+        <tbody>
+          ${versions.map((v: NodeVersion) => `<tr><td>${v.flowId}</td><td>${v.imageTag}</td><tr>`).join("")}
```

Comment:
Image tag is not set for external services.  How are you planning to support those services in the UI?

Reply:
That's a good call. Right now I will update this to default to "N/A" here and we can talk about what we might want to show in the future